### DISCUSSION TITLE
Upgrade to 9.11.2 - "Web Server Updated" Admin Log Messages

### DISCUSSION THREAD

**Discussion by james7342** [dnnsoftware/Dnn.Platform#5594] [General] [answered]
Not really an issue and more an observation. After upgrading 2 test sites to 9.11.2 I've noticed I'm getting "Web Server Updated" entires about every 5 mins. I know I can disable them etc. just wondering what in 9.11.2 would cause the new influx of this message. Looking in scheduler I can see "Monitor Web Servers " task is set to run every 5 mins. Not sure if this is due to https://github.com/dnnsoftware/Dnn.Platform/pull/5586 enhancement. 

Upgrades were from 9.10.2.
![screenshot_791](https://user-images.githubusercontent.com/27960723/228949145-07b22c36-e0d3-4423-8db2-e512866a2275.png)

**Comment by valadas** [comment]
It is a good assumption that it is related to that. @mitchelsellers @mtrutledge any thoughts on this?

**Comment by mtrutledge** [comment, marked as answer]
Yes it is related, the ServerController logs a info message any time a update happens to the Web Server table. This was existing functionality. It also used to only happen on startup because that is the only time the Servers were being looked at. When we updated the scheduler, we had to update the server's most recent activity, which causes an Informational log to be added. So this would be expected behavior. In future updates, we could remove the logging of the info in the Server Controller. 

Here is the Server Controller:
https://github.com/dnnsoftware/Dnn.Platform/blob/670658b5bddcb7604b740c3ae5eedae4864c90dd/DNN%20Platform/Library/Entities/Host/ServerController.cs#L151

**Reply by mitchelsellers** [reply]
If you are not on a multi-server environment or don't have concerns you can adjust the frequency of that job if the log entry clutter is of concern

**Reply by james7342** [reply]
Overall I'm not to worried about it personally and disabled in log settings at the moment. Was just curious as to why I was seeing this after upgrade when it wasn't there prior.

**Reply by james7342** [reply]
Maybe enhancement would be to check for multi server environments and schedule for every 5 mins else leave it the way it was prior. Not sure if that is an option or not. Or something along those lines as single server instances are now going to be flooded with these in the event logs as I think default log settings have this alert turned on.
